withDom.js

```diff
@@ -1,3 +1,5 @@
+require('raf/polyfill');
```

Comment:
Yeah, it seems like the `require('raf/polyfill')` should move inside the `if (!global.document)` since there is most likely a `requestAnimationFrame` already available when `global.document` exists.

For someone running tests in a browser the current code might make it look like react16 is more compatible with old browsers than it is.